apps/experiments/tasks.py

```diff
@@ -15,6 +15,13 @@
 from apps.utils.taskbadger import update_taskbadger_data
 
 
+@shared_task(bind=True, base=TaskbadgerTask)
+def async_create_experiment_version(self, experiment_id: int, *arg, **kwargs) -> dict:
```

Comment:
nit: specify the kwargs explicitly

Reply:
Addressed both of your comments (see the demo)